src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceEventGenerator.cs

```diff
@@ -69,8 +90,14 @@ private static void WriteEvent(LinuxAppServiceFileLogger logger, string evt)
             logger.Log(evt);
         }
 
+        private void WriteEvent(string eventPayload)
+        {
+            Console.WriteLine(eventPayload);
```

Comment:
@brettsam - is this needed if  `AzureFunctionsJobHost__Logging__Console__IsEnabled` is set to `true` ?

Reply:
Good question -- the Console Logger will write out logs with extra "stuff" around the message itself (like `Info:`). I'm assuming that Antares will use a regex to identify these logs -- do we know if that would match?

Reply:
@brettsam - I assume that the behavior of ConsoleLogger would be the same on the Linux Consumption plan, as in [LinuxContainerEventGenerator](https://github.com/Azure/azure-functions-host/blob/14adf6d336db5da0feb0221cc4fe0a40bc14edb0/src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs)?
@pragnagopa or @divyagandhisethi - do you know how Antares handles the stdout for Linux Consumption?

Reply:
I think the log format depends on the logging driver used ( the one used in Antares might not be the same as what is used on Linux Consumption but I think they are the same currently also Antares never sees any of the raw logs from Linux Consumption as it is handled elsewhere)

In this case as long as the eventual log format emitted by the containers are parsed correctly by the log parsers used in Antares everything should be good. If a particular log entry doesnt match the format the parsers expect then it will be dropped and wont reach AzureMonitor. Testing this on a Linux worker will help confirm/clarify things greatly :)

Reply:
Ideally we do not have to write to console directly. I will track this as a separate issue to remove `cosole.writeline` across linux skus and set `AzureFunctionsJobHost__Logging__Console__IsEnabled` and test them out. resolving this comment.